python/utils.py

```diff
@@ -677,6 +677,11 @@ def fcnRegexp(pattern, string):
             break
     if not found:
         raise RuntimeError("Cannot find any suitable spatialite module")
+    if any(['.gpkg' in arg for arg in args]):
+        try:
+            cur.execute("SELECT EnableGpkgAmphibiousMode()")
+        except:
+            warnings.warn("Could not enable geopackage amphibious mode")
```

Comment:
Can you align it with https://github.com/qgis/QGIS/blob/3be17e5192898476a0e385fc3f3c331ac1d50512/python/utils.py#L69 please?

Reply:
Sure thing, though I changed the formatwarning to simply using warning, I doubt it was necessary to write to a file, but let me know if I should use formatwarning instead.

Reply:
Can you do an experiment for me on the console please?

```
test = warnings.warn('Repeat with me: I will stick to the standard way of doing things in QGIS and prefer the way existing code is doing it.')
print(test)
```

Thank you.